Core/automation/lib/python/core/date.py

```diff
@@ -40,6 +40,11 @@
     JodaDateTime = None
     JodaDateTimeZone = None
 
+try:
+    from org.openhab.core.library.items import DateTimeItem
+except:
+    DateTimeItem = None
+
```

Comment:
Perfect! Lets move this into the try block below to save space. Also we need to keep backwards compatibility with OH2.x so we need to end up with:

```python
try:
    # OH2.x compat1x or OH3
    from org.openhab.core.library.types import DateTimeType
    from org.openhab.core.library.items import DateTimeItem
except:
    DateTimeType = None
    DateTimeItem = None

try:
    from org.eclipse.smarthome.core.library.types import DateTimeType as EclipseDateTimeType
    from org.eclipse.smarthome.core.library.items import DateTimeItem as EclipseDateTimeItem
except:
    EclipseDateTimeType = None
    EclipseDateTimeItem = None
```